### PR TITLE
Bump version.

### PR DESCRIPTION
**Pull request by benjaminrwilson**
Checklist
* [x] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [x] Reset the build number to `0` (if the version changed)
* [x] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [x] Ensured the license file is being packaged.

### CODE REVIEW

**Comment by benjaminrwilson** [comment]
@mdraw, ready for review -- whenever you get a chance.

File: recipe/meta.yaml [hunk 1]
```diff
@@ -1,29 +1,32 @@
...
+  sha256: 5a50c0eeadcad4324a6bd895b6dcbe17068558143c8080e1cc6cdbe637c679de
 
 build:
   noarch: python
-  number: 1
-  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+  script: 
+    - {{ PYTHON }} setup.py antlr
+    - {{ PYTHON }} -m pip install . -vv
```
Comment (omry) [inline recipe/meta.yaml:17]: As far as I know, pip install should run antlr. Did did you see otherwise?

File: recipe/meta.yaml [hunk 2]
```diff
@@ -1,29 +1,32 @@
...
     - antlr ==4.8
   host:
-    - python >=3.6,<3.10
+    - python >=3.6
     - pip
   run:
-    - python >=3.6,<3.10
-    - omegaconf >=2.0.5,<2.1
+    - python >=3.6
+    - omegaconf
```
Comment (omry) [inline recipe/meta.yaml]: Please keep the same version constraints as Hydra (Any OmegaConf 2.1 version):

https://github.com/facebookresearch/hydra/blob/437e20df9f0f3cb671e117c382f2dee727afbbf2/requirements/requirements.txt#L1

File: recipe/meta.yaml (hunk 1 above)
Comment (benjaminrwilson) [reply recipe/meta.yaml:17]: This was failing without line 16. I'll remove it to double check.

Comment (benjaminrwilson) [reply recipe/meta.yaml:17]: @omry https://dev.azure.com/conda-forge/feedstock-builds/_build/results?buildId=349029&view=logs&j=656edd35-690f-5c53-9ba3-09c10d0bea97&t=e5c8ab1d-8ff9-5cae-b332-e15ae582ed2d&l=797

Comment (omry) [reply recipe/meta.yaml:17]: Odd. what command is it running?
I never need to run the antlr command directly locally, `pip install .` do the trick.

Comment (benjaminrwilson) [reply recipe/meta.yaml:17]: I'm not sure outside of what is shown in the logs. Do you think it will cause an issue?

**Comment by benjaminrwilson** [comment]
@conda-forge-admin, rerender

**Comment by benjaminrwilson** [comment]
@omry, @hadim, @mdraw, what do you all think about the state of this?

File: recipe/meta.yaml (hunk 1 above)
Comment (omry) [reply recipe/meta.yaml:17]: No, it should be okay to trigger it manually.

**Review by omry — APPROVED** [review]
Overall lgtm.

**Comment by benjaminrwilson** [comment]
@hadim, @mdraw would you all mind taking a look / merging if you're okay with the PR?

**Review by hadim — APPROVED** [review]

**Comment by hadim** [comment]
LGTM.

**Pull request merged**